services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/internal/converter/ConverterUtils.java

```diff
@@ -128,55 +129,67 @@ private static int findTrimExclusiveEnd(String string, char characterToTrim, int
         }
     }
 
-    public static String[] chunk(String valueToChunk, int... splitSizes) {
-        String[] result = new String[splitSizes.length + 1];
-        int splitStartInclusive = chunkLeft(valueToChunk, result, splitSizes);
+    /**
+     * Chunk the String without nanoseconds into a String array, and validates the format of input string and output result.
+     * The negative sign is stored in the first element with 'year', if applicable.
+     * @param valueToChunk
+     * @return String array storing chunked pieces of Date and Time
+     */
+    public static String[] chunkDateTime(String valueToChunk) {
```

Comment:
Can we not write our own date time parsing? Can we not use `DateTimeFormatter` for this?